### DISCUSSION TITLE
C++ Turbo Native Modules

### DISCUSSION THREAD

**Discussion by Truong-coder** [reactwg/react-native-new-architecture#114] [Q&A]
Greetings React Native community,
I have been working on RN new architecture for 2 weeks. I'm following instructions on the official doc and I'm struggling with this C++ turbo native module on C++ section. https://reactnative.dev/docs/the-new-architecture/cxx-cxxturbomodules 
I thoroughly follow the instructions but not be able to run the code. I run the npx react-native start but the screen is empty and return no error, sometimes it return the EPERM error, operation not permitted but I'm be able to fix this. I can't debug with react tool as well.

If anyone could make it works please help me. Thank you!

**Comment by christophpurrer** [comment]
@Truong-coder  > Happy to help here. 
Could you share more information about your particular problem?
Have you tried it on Android or iOS or both?
What's the particular error message?
Can you share your draft / example via GitHub branch - in case you started on a new (sample) app to share

**Reply by Truong-coder** [reply]
Thanks for your response. it turns out I copied the code in the wrong place. I fix it and it works just fine.

**Reply by Truong-coder** [reply]
But i still struggle with the EPERM error!
[image_2023_02_10T06_42_39_571Z-1.png](https://user-images.githubusercontent.com/81181191/219706781-23052088-f731-4458-9288-f7ab49c699a5.png)

**Reply by Truong-coder** [reply]
Do you familiar with this error?
